notebooks/254-llm-chatbot/254-rag-chatbot.ipynb

```diff
@@ -83,7 +83,7 @@
     "\"accelerate\"\\\n",
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #5.        value=llm_model_ids[-1],
</pre><p>could you please change default value on llm_model_ids[0] instead of -1?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/openvinotoolkit/openvino_notebooks/pull/1824/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='7'/>

Reply:
The reason for using [-1] instead of [0] is that the default model for chatbot can do RAG but not good at it, so we have to put the default RAG on then end of this list. For example , ting-llama can support RAG, but its accuracy of RAG is not good enough, so neural-chat should be the default one.

Reply:
got it